src/NumberGuessingGame.js

```diff
@@ -13,65 +13,43 @@ function getRandomNumber() {
 
 const MAX_ATTEMPTS = 5;
 
-class NumberGuessingGame extends Component {
-  constructor(props) {
-    super(props);
+const NumberGuessingGame = () => {
+  const [numberToGuess, setNumberToGuess] = useState(getRandomNumber());
```

Comment:
This works for the most part but `numberToGuess` is actually initialized twice.

Try to figure out a way to initialize it once 👍

Reply:
I looked over my code but I only saw numberToGuess initialized once through the useState hook. I am not sure if I overlooked something?